src/components/WelcomeToSDC.tsx

```diff
@@ -73,21 +77,25 @@ export const WelcomeToSDC: React.FC<WelcomeToSDCProps> = ({ session }) => {
                                 )}
```

Comment:
For the button, please add a breakpoint for width on line 60, (change on line 70)

```typescript
                                        <Button
                                            color="white"
                                            backgroundColor={
                                                colourTheme.colors.Blue
                                            }
                                            _hover={{
                                                backgroundColor:
                                                    colourTheme.colors
                                                        .LightBlue,
                                            }}
                                            width={{ base: "100%", md: "50%" }}
                                            borderRadius="6px"
                                            fontWeight={"200"}
                                        >
                                            {t("home.registerNow")}
                                        </Button>
```

![image](https://user-images.githubusercontent.com/44826218/142134639-aa51d0d6-d86f-4aac-8cf4-67212c9bf48c.png)